acceptance/setup/git/pre-suite/010_TestSetup.rb

```diff
@@ -1,85 +1,76 @@
 test_name "Install packages and repositories on target machines..." do
-require 'beaker/dsl/install_utils'
-extend Beaker::DSL::InstallUtils
+  require 'beaker/dsl/install_utils'
+  extend Beaker::DSL::InstallUtils
 
-  SourcePath  = Beaker::DSL::InstallUtils::SourcePath
-  GitURI      = Beaker::DSL::InstallUtils::GitURI
   GitHubSig   = Beaker::DSL::InstallUtils::GitHubSig
 
-  tmp_repositories = []
-  options[:install].each do |uri|
-    raise(ArgumentError, "Missing GitURI argument. URI is nil.") if uri.nil?
-    raise(ArgumentError, "#{uri} is not recognized.") unless(uri =~ GitURI)
-    tmp_repositories << extract_repo_info_from(uri)
-  end
+  gem_source = ENV["GEM_SOURCE"] || "https://rubygems.org"
+  sha = ENV['SHA'] || `git rev-parse HEAD`.chomp
+  repository = extract_repo_info_from(build_git_url("puppet"))
 
-  repositories = order_packages(tmp_repositories)
+  gemfile_contents = <<END
+source '#{gem_source}'
+gem '#{repository[:name]}', :git => '#{repository[:path]}', :ref => '#{sha}'
```

Comment:
This requires that any changes you want to test have been pushed out to github, correct?

Reply:
Not sure, I think `repository[:path]` should include your local server if one was specified, and so should work. Note this code was moved from below, so shouldn't be any functional changes.